src/include/hip/rpp_hip_common.hpp

```diff
@@ -1713,6 +1713,34 @@ __device__ __forceinline__ void rpp_hip_math_multiply24_const(d_float24 *src_f24
     dst_f24->f4[5] = src_f24->f4[5] * multiplier_f4;
 }
 
+// d_float8 bitwiseAND
+
+__device__ __forceinline__ void rpp_hip_math_bitwiseAnd8(d_float8 *src1_f8, d_float8 *src2_f8, d_float8 *dst_f8)
+{
+        dst_f8->f1[0] = (float)((uchar)(src1_f8->f1[0]) & (uchar)(src2_f8->f1[0]));
```

Comment:
why  can't we do it as vector instructions?

Reply:
HIP CUDA doesn't support vectorized 8 pixel instruction for Bitwise operations. Thus used pixel-wise operations.